src/backend/login/usingPassport.js

```diff
@@ -1,36 +1,36 @@
-const passport = require('passport');
-const OneLoginStrategy = require('passport-openid').Strategy;
+const saml = require('passport-saml');
 require('../lib/config');
+const fs = require('fs');
+const path = require('path');
 
-/* Set environmental variables for these
-https://www.onelogin.com/ to create your account.
-Tutorial is here: https://developers.onelogin.com/quickstart/authentication/nodejs
-OIDC_BASE_URI=https://openid-connect.onelogin.com/oidc
-OIDC_CLIENT_ID=ddf97d-xxx-xxxx-e92920
-OIDC_CLIENT_SECRET=f021f63bxxxx71f2c6a90c9b
-OIDC_REDIRECT_URI=http://localhost:3000/oauth/callback
+/*
+Tutorial is here: https://medium.com/disney-streaming/setup-a-single-sign-on-saml-test-environment-with-docker-and-nodejs-c53fc1a984c9
+SAML2_BASE_URI=https://openid-connect.onelogin.com/oidc
+SAML2_CLIENT_ID=ddf97d-xxx-xxxx-e92920
+SAML2_CLIENT_SECRET=f021f63bxxxx71f2c6a90c9b
+SAML2_REDIRECT_URI=http://localhost:3000/oauth/callback
 */
 
-// Configure the OpenId Connect Strategy
-// with credentials obtained from OneLogin
-const { OIDC_BASE_URI, OIDC_CLIENT_ID, OIDC_CLIENT_SECRET, OIDC_REDIRECT_URI } = process.env;
+const { SAML2_BASE_URI, SAML2_REDIRECT_URI } = process.env;
```

Comment:
Do you need to check for, and log errors if any of these are missing or in the wrong format?  Same for all the other env variables below.  It seems like getting this right could be tricky for someone without full knowledge, can you help a dev get this right via docs, error logs, errors, examples in the `env.sample`, etc?